Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "linux-loader"
+version = "0.1.0"
```

Comment:
@andreeaflorescu is this correct starting version number?

Reply:
I would have it as 0.0.1 and when we publish it to crates.io I would switch to 0.1.0. But it doesn't matter that much if we don't update the version on following PRs.